digitalforms-client/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.7.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>ca.bc.gov.open</groupId>
+	<artifactId>digitalforms-client</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>digitalforms-client</name>
+	<description>Java client for RoadSafetyBC DigitalForms database</description>
```

Comment:
Possibly change this to reflect that this is an ORDS client for the digital forms project?